tutorials/shaders/shader_reference/spatial_shader.rst

```diff
@@ -141,6 +141,19 @@ Users can override the modelview and projection transforms using the ``POSITION`
 the value from ``VERTEX`` is ignored and projection does not happen. However, the value passed to the fragment shader
 still comes from ``VERTEX``.
 
+To access the camera direction to the object in view space the following code can be used:
+
+.. code-block:: glsl
+    
+    render_mode skip_vertex_transform;
+    void vertex() {
+        vec3 camera_direction  = (MODELVIEW_MATRIX * vec4(VERTEX, 1.0)).xyz;
```

Comment:
This is just vertex position in Camera space (which is equivalent to camera direction to vertex), it is not camera direction to object. Camera direction to object is typically ``(MODELVIEW_MATRIX * vec4(0.0, 0.0, 0.0 1.0)).xyz;`` when the mesh is centered about the origin. 

I don't know if this is a really helpful example.

Reply:
My original code was actually this: `normalize(WORLD_MATRIX[3].xyz - CAMERA_MATRIX[3].xyz) = Camera direction to object in world space` but Lyuma mentioned that it would be confusing since it would be too specific for a billboard shader

Reply:
I think it fits in fine with the other examples below. As long as they are framed with "here is how you can calculate some common variables using just the built ins"